src/page/Switch.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import routeResolver from '../routing/routeResolver.js';
+import matcher from '../routing/matcher.js';
+
+import { URL_MATCH_FAILURE } from '../actions.js';
+
+class Switch extends React.Component {
```

Comment:
it is switching pages. IMO it's described in docs and its okay

Reply:
I would still like to have it called PageSwitch (or something similar). Plus I'd like the project structure to be src/ComponentName/ComponentName.js or if component doesn't have any helpers an additional files it could be placed in src without specific catalog, like src/ComponentName.js. That's a common approach which is good in my opinion.